src/static/js/move_adopt.js

```diff
@@ -80,6 +82,43 @@ $(function() {
         },
 
         bind: function() {
+            if($("#eventlink").is(":checked"))
+                $("#event").closest("tr").fadeIn();
+            else
+                $("#event").closest("tr").fadeOut();
+
+            //callback when eventlink changed its status
+             $("#eventlink").change(function(){
+                // event link needs a movement date
+                if (this.checked && $("#movementdate").val() == ""){
+                    validate.notblank([ "movementdate" ]);
+                    tableform.dialog_error(_("Fill out adoption date before linking to event."));
+                    this.checked = false;
+                }
+                $("#event").empty();
+                if (this.checked){
+                    $("#event").closest("tr").fadeIn();
+                    move_adopt.event_dates();
+                }
+                else
+                    $("#event").closest("tr").fadeOut();
+
+            });
+            //callback when movementdate is changed
+            $("#movementdate").change(function(){
+                $("#event").empty();
```

Comment:
Why not hide the event fields until movement date is completed?

Reply:
@bobintetley we thought about this  but it would be odd - the page starts with a value (today) in the movement date field, but the value can be deleted (which would make the entire movement invalid), and it would be weird if the event would hide and show depending on whether there’s a movement date value. What do you think?